pkg/envtest/webhook.go

```diff
@@ -147,6 +147,8 @@ func (o *WebhookInstallOptions) PrepWithoutInstalling() error {
 
 // Install installs specified webhooks to the API server.
 func (o *WebhookInstallOptions) Install(config *rest.Config) error {
+	defaultWebhookOptions(o)
```

Comment:
Why was this added?

Reply:
Is this something where the behavior changed because of the k8s bump? Or is this an unrelated bug fix? If the latter, should we split it out into a separate PR?

Reply:
Good question. `options.MaxTime` is handled differently in `poll.Immediate()`:

```
// closed. A timeout of 0 is interpreted as an infinity, and in such a case
// it would be the caller's responsibility to close the done channel.
// Failure to do so would result in a leaked goroutine.
```

without `defaultWebhookOptions()`, `MaxTime` is zero. If "inifinity" is not wanted, this change could be moved to a separate PR.

Reply:
Created #2289, closing.